### PR TITLE
Add more capabilities to the detault set used by helm template

### PR DESCRIPTION
**Pull request by mattfarina**
- These are the capabilities as of k8s 1.14
- They are generated because getting them at runtime uses so much
  memory it causes CI errors

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [x] this PR contains unit tests
- [x] this PR has been tested for backwards compatibility

### CODE REVIEW

File: pkg/chartutil/capabilities.go [hunk 1]
```diff
@@ -69,3 +69,55 @@ func (v VersionSet) Has(apiVersion string) bool {
 	_, ok := v[apiVersion]
 	return ok
 }
+
+// Default versions as of 1.14, excluding deprecated versions, focused on workloads
+func defaultVersions() []string {
```
Comment (bacongobbler) [inline pkg/chartutil/capabilities.go]: Is there a lib in kubernetes that lists all of the default API groups? That way we wouldn't have this large list of default API versions laying around.

Comment (mattfarina) [reply pkg/chartutil/capabilities.go]: I'll poke around and see

Comment (mattfarina) [reply pkg/chartutil/capabilities.go]: Looks like there is an API for that and it provides 554 items to stick in the default set.

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
looks good to merge once CI passes! Thanks for finding `scheme.Scheme.AllKnownTypes()`.

**Comment by mattfarina** [comment]
@bacongobbler I re-wrote it to get around the runtime memory issues. Uses generation now.

File: pkg/chartutil/generator/capabilities_default_versions_generate.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+			check[gv] = struct{}{}
+			s = append(s, gv)
+		}
+		if _, ok := check[gvk]; !ok {
+			check[gvk] = struct{}{}
+			s = append(s, gvk)
+		}
+	}
+
+	return s
```
Comment (bacongobbler) [inline pkg/chartutil/generator/capabilities_default_versions_generate.go]: small nit, but would it be worth it to sort the string slice before returning here? That way the string slice is consistent between `go generate` invocations.

**Review by bacongobbler — APPROVED** [review]
Code LGTM, just one comment about a potential enhancement to the generated code output

File: pkg/chartutil/generator/capabilities_default_versions_generate.go (hunk 2 above)
Comment (mattfarina) [reply pkg/chartutil/generator/capabilities_default_versions_generate.go]: Great idea. sort added.

**Review by bacongobbler — APPROVED** [review]

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @mattfarina. Good to merge.

**Pull request merged**